sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -21,7 +21,7 @@ grammar SqlBase;
    * When false, INTERSECT is given the greater precedence over the other set
    * operations (UNION, EXCEPT and MINUS) as per the SQL standard.
    */
-  public boolean legacy_setops_precedence_enbled = false;
+  public boolean legacy_setops_precedence_enabled = false;
```

Comment:
Please split this renaming as an another small PR before this. That will be helpful for all reviewers/developers with a clear commit history. Although this might be considered minor, a PR with one theme is always the best.
cc @gatorsmile